tests/Doctrine/Tests/DBAL/Functional/Ticket/DBAL630Test.php

```diff
@@ -170,4 +171,12 @@ public static function booleanTypeConversionWithoutPdoTypeProvider() : iterable
             [null, null],
         ];
     }
+
+    private function getWrappedConnection() : PDOConnection
+    {
+        $connection = $this->connection->getWrappedConnection();
+        self::assertInstanceOf(PDOConnection::class, $connection);
+
+        return $connection;
+    }
```

Comment:
Have you considered redeclaring the property like this?

```php
/** @var PDOConnection */
protected $connection;
```

Reply:
`protected $connection` is always the wrapper connection. It's the wrapped connection class that we need to make an assertion about.

Reply:
Oh right sorry!